homeassistant/components/openai_conversation/__init__.py

```diff
@@ -63,15 +71,25 @@ async def async_process(
         self, user_input: conversation.ConversationInput
     ) -> conversation.ConversationResult:
         """Process a sentence."""
-        model = DEFAULT_MODEL
+        raw_prompt = self.entry.options[CONF_PROMPT]
+        raw_contprompt = self.entry.options[CONF_CONTINUED_PROMPT]
+        engine = self.entry.options[CONF_ENGINE]
+        max_tokens = self.entry.options[CONF_MAX_TOKENS]
+        top_p = self.entry.options[CONF_TOP_P]
+        temperature = self.entry.options[CONF_TEMPERATURE]
+        template_variables = {
+            "matched_areas": self._matching_areas(user_input.text),
+            "matched_entities": self._matching_entities(user_input.text),
```

Comment:
let's remove this. This is making the integration overly complicated. If a user wants to limit areas etc that are send, they can define specific areas in the prompt.

Reply:
Removed